src/apps/loyalty/containers/3w_thank_you/index.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react"
+import styled from "styled-components"
+
+import colors from "../../../../assets/colors"
+import Button from "../../../../components/buttons/inverted"
```

Comment:
You can do the same as here: https://github.com/artsy/reaction-force/pull/86/files/953411416e68dd4bf1451407be0d98c564cf817f#diff-5ca228937ab8f58c41d6ae3aa13668e7R5